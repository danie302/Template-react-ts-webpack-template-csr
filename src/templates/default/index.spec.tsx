import React from "react";
import { shallow } from "enzyme";

import Default from "./index";

it("renders the layout", () => {
    const result = shallow(<Default />);
    expect(result).toMatchSnapshot();
});
