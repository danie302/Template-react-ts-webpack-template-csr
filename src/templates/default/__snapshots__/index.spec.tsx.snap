// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the layout 1`] = `ShallowWrapper {}`;
